paddle/operators/save_load_combine_op_test.cc

```diff
@@ -0,0 +1,125 @@
+  attrs.insert({"position_counter", static_cast<int>(pos_counter)});
+  auto save_combine_op = paddle::framework::OpRegistry::CreateOp(
+      "save_combine", {{"X", {var_name}}}, {}, attrs);
+  save_combine_op->Run(scope, place);
+  return expect;
+}
+
+int* create_and_run_load_combine_op(std::string var_name, int pos_counter,
+                                    std::string filename,
+                                    paddle::framework::Scope& scope,
+                                    paddle::platform::CPUPlace& place,
+                                    paddle::framework::LoD& actual_lod) {
+  auto load_var = scope.Var(var_name);
+  auto target = load_var->GetMutable<paddle::framework::LoDTensor>();
+  paddle::framework::AttributeMap attrs;
+  attrs.insert({"file_path", std::string(filename)});
+  attrs.insert({"position_counter", static_cast<int>(pos_counter)});
+  auto load_combine_op = paddle::framework::OpRegistry::CreateOp(
+      "load_combine", {}, {{"Out", {var_name}}}, attrs);
+  load_combine_op->Run(scope, place);
+  int* actual = target->data<int>();
+  actual_lod = target->lod();
+  return actual;
+}
+
+void check_values(int* expect, int* actual, paddle::framework::LoD expect_lod,
+                  paddle::framework::LoD actual_lod, const int& numel) {
+  for (int64_t i = 0; i < numel; ++i) {
+    EXPECT_EQ(expect[i], actual[i]);
+  }
+  EXPECT_EQ(expect_lod.size(), actual_lod.size());
+  for (size_t i = 0; i < expect_lod.size(); ++i) {
+    for (size_t j = 0; j < expect_lod[i].size(); ++j) {
+      EXPECT_EQ(expect_lod[i][j], actual_lod[i][j]);
+    }
+  }
+}
+
+// Here, we create 4 LoDTensors and use save_combine_op to first save these
+// in a single file. Then, we use load_combine_op to load these sequentially
+TEST(SaveLoadCombineOp, CPU) {
+  paddle::framework::Scope scope;
+  paddle::platform::CPUPlace place;
+
+  std::vector<int> lod1 = {0, 1, 2, 3};
+  int numel1 = 100;
+  paddle::framework::LoD expect_lod1;
+  int* expect1 = create_and_run_save_combine_op(
+      10, 10, lod1, "test_var1", 0, "tensor.save", scope, place, expect_lod1);
+
+  std::vector<int> lod2 = {0, 2, 5};
+  int numel2 = 200;
+  paddle::framework::LoD expect_lod2;
+  int* expect2 = create_and_run_save_combine_op(
+      10, 20, lod2, "test_var2", 1, "tensor.save", scope, place, expect_lod2);
+
+  std::vector<int> lod3 = {0, 2, 3};
+  int numel3 = 4000;
+  paddle::framework::LoD expect_lod3;
+  int* expect3 = create_and_run_save_combine_op(
+      200, 20, lod3, "test_var3", 2, "tensor.save", scope, place, expect_lod3);
+
+  std::vector<int> lod4 = {0, 1};
+  int numel4 = 1000;
+  paddle::framework::LoD expect_lod4;
+  int* expect4 = create_and_run_save_combine_op(
+      50, 20, lod4, "test_var4", 3, "tensor.save", scope, place, expect_lod4);
+
+  paddle::framework::LoD actual_lod1, actual_lod2, actual_lod3, actual_lod4;
+  int* actual1 = create_and_run_load_combine_op("out_var1", 0, "tensor.save",
```

Comment:
When LoadCombineOp loads all parameters at the same time, line 112 maybe like:
`create_and_run_load_combine_op("out_var_list", "tensor.save", scope, place)` ?